_posts/2019-08-12-how-to-configure-production-grade-aws-account-structure.adoc

```diff
@@ -1463,12 +1540,18 @@ The example code above does the following:
 . **Create IAM users**. For this example, we create `alice` and `bob`, and carol, adding `alice` to the `full-access`
   IAM group and `carol` to the `billing` IAM group. _Note_: your own IAM user (the one you created manually) should be
   in the `users` list; we'll use the `import` command to put this user under Terraform management shortly.
+# TODO: maybe the above works, to use `import` to put the user under Terraform management, but it was such a long frickin' section that I decided to skip the import. Instead, I used aws-vault to run the first terraform apply using the root account user, and then changed it to use the root IAM user once they were created by Terraform.
```

Comment:
This isn't really a recommended approach though. Best security practices state that you should **never** do anything with the root user other than to create an admin IAM user, let alone have credentials for them in a CLI tool. We should emphasize this somewhere in here to discourage users from taking this approach, and that despite this being a long approach, you want to follow it.

Reply:
I ended up having to use `import`. I should remove this line!